templates/components/api-server/template/server/server.js

```diff
@@ -3,20 +3,23 @@ var boot = require('loopback-boot');
 
 var app = module.exports = loopback();
 
+// boot scripts mount components like REST API
+boot(app, __dirname);
+
```

Comment:
Please revert this change, as it is effectively removing support for async boot scripts.